way_to_home/custom_user/models.py

```diff
@@ -27,6 +28,30 @@ def to_dict(self):
             'phone_number': self.phone_number
         }
 
+    def update(self, password=None, google_token=None, phone_number=None, is_active=None):
+        """Method for object update."""
+        if password:
+            self.password = password
```

Comment:
```suggestion
            self.password = password
```
You need to use `.set_password()` method to set the password properly. In your case, you save password just a string.

Reply:
ok, done